### PR TITLE
Eip github bot

### PR DESCRIPTION
**Pull request by angie1148**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by MicahZoltu** [comment]
@alita-moore Why is the bot posting an empty list of issues?

File: Gemfile.lock [hunk 1]
```diff
@@ -229,6 +230,7 @@ GEM
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (3.1.1)
+    racc (1.5.2)
```
Comment (GregTheGreek) [inline Gemfile.lock:233]: FWII this is not sketchy, it was racc was explicitly added is a dependency of nokogiri in [this](https://nokogiri.org/CHANGELOG.html#notes) release.

Comment (MicahZoltu) [reply Gemfile.lock:233]: Any idea about the other version updates?

Comment (GregTheGreek) [reply Gemfile.lock:233]: They're just minor bumps. I'd say we don't **need** to bump but some of the deps had security patches, nothing serious, but also nothing drastic.

**Comment by GregTheGreek** [comment]
> @alita-moore Why is the bot posting an empty list of issues?

Looked into this. It's unrelated to the ruby changes because the merge bot is managed via a nodejs app in GitHub actions. The ruby code is run in travis-ci and is used to serve the website.

**Comment by alita-moore** [comment]
Yeah it's most likely that it's trying to post the error message (new feature) but when accessing the error error.message is undefined and therefore nothing. I'll add that edgecase

**Comment by MicahZoltu** [comment]
@angie1148 Can you resolve the conflicts with this so we can get it merged?

File: Gemfile.lock (hunk 1 above)
Comment (GregTheGreek) [reply Gemfile.lock:233]: Ah wait, let me see if we should bump racc to the same versions.

**Comment by alita-moore** [comment]
the bot now supports multiple files 🎉

**Comment by lightclient** [comment]
@angie1148 what is this PR accomplishing?

**Comment by MicahZoltu** [comment]
IIRC, it is fixing a security issue reported by dependabot.

**Comment by GregTheGreek** [comment]
> IIRC, it is fixing a security issue reported by dependabot.

I dont have access to see if it was a security update, but when I previously reviewed the gemfile changes the version bumps were fine. There was a minor security patch applied (IIRC some regex parsing bugs) but overall if the patch numbers are the same lgtm.

**Comment by MicahZoltu** [comment]
This still has conflicts by the way.

**Comment by lightclient** [comment]
I'm going to close. If someone would like to continue pursuing, feel free to reopen.

**Pull request closed without merging**